common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -1,113 +1,121 @@
-            });
+        DiscussionSpecHelper.setUpGlobals = function() {
+            DiscussionUtil.loadRoles(DiscussionSpecHelper.getTestRoleInfo());
+            window.$$course_id = 'edX/999/test';
+            window.user = new DiscussionUser(DiscussionSpecHelper.getTestUserInfo());
             return DiscussionUtil.setUser(window.user);
         };
 
-        DiscussionSpecHelper.makeTA = function () {
-            return DiscussionUtil.roleIds["Community TA"].push(parseInt(DiscussionUtil.getUser().id));
+        DiscussionSpecHelper.getTestUserInfo = function() {
+            return {
+                username: 'test_user',
+                id: '567',
+                upvoted_ids: []
+            };
+        };
+
+        DiscussionSpecHelper.getTestRoleInfo = function() {
+            return {
+                Moderator: [],
+                Administrator: [],
+                'Community TA': []
+            };
         };
 
-        DiscussionSpecHelper.makeModerator = function () {
-            return DiscussionUtil.roleIds.Moderator.push(parseInt(DiscussionUtil.getUser().id));
+        DiscussionSpecHelper.makeTA = function() {
+            return DiscussionUtil.roleIds['Community TA'].push(parseInt(DiscussionUtil.getUser().id, 10));
         };
 
-        DiscussionSpecHelper.makeAjaxSpy = function (fakeAjax) {
-            return spyOn($, "ajax").and.callFake(function (params) {
+        DiscussionSpecHelper.makeModerator = function() {
+            return DiscussionUtil.roleIds.Moderator.push(parseInt(DiscussionUtil.getUser().id, 10));
+        };
+
+        DiscussionSpecHelper.makeAjaxSpy = function(fakeAjax) {
+            return spyOn($, 'ajax').and.callFake(function(params) {
                 fakeAjax(params);
                 return {
-                    always: function () {
+                    always: function() {
                     }
                 };
             });
         };
 
-        DiscussionSpecHelper.makeEventSpy = function () {
+        DiscussionSpecHelper.makeEventSpy = function() {
             return jasmine.createSpyObj('event', ['preventDefault', 'target']);
         };
 
-        DiscussionSpecHelper.makeCourseSettings = function (is_cohorted) {
-            if (typeof is_cohorted === 'undefined' || is_cohorted === null) {
-                is_cohorted = true;
-            }
+        DiscussionSpecHelper.makeCourseSettings = function() {
             return new DiscussionCourseSettings({
                 category_map: {
                     children: ['Test Topic', 'Other Topic'],
                     entries: {
                         'Test Topic': {
-                            is_cohorted: is_cohorted,
+                            is_cohorted: true,
                             id: 'test_topic'
                         },
                         'Other Topic': {
-                            is_cohorted: is_cohorted,
+                            is_cohorted: true,
```

Comment:
I searched `edx-solutions:master` for calls to `makeCourseSettings`, and none of them passed an `is_cohorted` parameter.  The original code defaulted to `true`, so it looked fine to accept this upstream version.